releasewarrior/templates/firefox/release-rc.json.tmpl

```diff
@@ -37,13 +37,13 @@
             {
                 "alias": "mirrors",
                 "description": "pushed to mirrors/releases",
-                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/wiki/Release-Promotion-Tasks#push-artifacts-to-releases-directory",
+                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/docs/release-promotion/desktop/howto.md#push",
```

Comment:
URL seems wrong, http:// in the middle of the url. Wrong interpolation/search-replace maybe?